maintainers/maintainer-list.nix

```diff
@@ -12032,6 +12032,16 @@
       fingerprint = "ADF4 C13D 0E36 1240 BD01  9B51 D1DE 6D7F 6936 63A5";
     }];
   };
+  silasdavis = {
```

Comment:
I had this lying around and thought I'd add my contact details, not sure if I should be a maintainer for any jetbrains stuff just because of this one change, but I expect to add some other packages so I thought I'd commit this now. Obviously I can pull it out if that is cleaner.